### PR TITLE
QA-543: Fix `test_dbus_*` tests flodding mail server and CI logs

### PR DESCRIPTION
**Pull request by lluiscampos**
Read the commit descriptions for details :popcorn:

### CODE REVIEW

**Comment by lluiscampos** [comment]
@mender-test-bot start pipeline

**Review by merlin-northern — APPROVED** [review]
left some minor comments. thanks! looks nice.

File: tests/tests/test_monitor_client.py [hunk 1]
```diff
@@ -1057,13 +1035,31 @@ def test_dbus_pattern_match(self, monitor_commercial_setup_no_client):
...
+            mender_device.run(
+                "dbus-send --system --dest=io.mender.AuthenticationManager /io/mender/AuthenticationManager io.mender.Authentication1.FetchJwtToken"
+            )
+            mail, messages = get_and_parse_email(
+                monitor_commercial_setup_no_client, user_name
+            )
+            if len(messages) > 0:
+                break
+            logger.info("test_dbus_pattern_match: got no alerts, retrying in 30s")
+            time.sleep(30)
```
[the remark this answers is not in the document]
Comment (lluiscampos) [reply tests/tests/test_monitor_client.py:1058]: This one I think I will leave as-is.

It is two magic numbers here that we would need to set, the retries (10) and sleep between retries (30). The comment above explains where the numbers come from and if we parametrize this one we should also parametrize `get_and_parse_email_n`.

Ultimately, this is totally arbitrary and as I pointed out in the comment and Git message a result of a buggy feature that we should consider remove instead.

**Comment by lluiscampos** [comment]
@mender-test-bot start pipeline

**Pull request merged**